### PR TITLE
chore: fix a large number of spelling issues

### PR DESCRIPTION
**Pull request by tinyfoolish**
# Description

fix a large number of spelling issues

# How Has This Been Tested?

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [x] Tested via GitHub Actions

## Summary by CodeRabbit

* **Documentation**
  * Corrected minor spelling and grammar errors in code comments throughout the codebase.

### CODE REVIEW

File: cmd/zetatool/cli/cctx_tracker.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func trackCCTX(ctx *zetatoolcontext.Context) (*cctx.TrackingDetails, error) {
 		cctxTrackingDetails = cctx.NewTrackingDetails()
 		err                 error
 	)
-	// Get the ballot identifier for the inbound transaction and confirm that cctx status in atleast either PendingInboundConfirmation or PendingInboundVoting
+	// Get the ballot identifier for the inbound transaction and confirm that cctx status in at least either PendingInboundConfirmation or PendingInboundVoting
```
Comment (lumtis) [inline cmd/zetatool/cli/cctx_tracker.go]:
```suggestion
	// Get the ballot identifier for the inbound transaction and confirm that cctx status is at least either PendingInboundConfirmation or PendingInboundVoting
```

File: e2e/e2etests/test_bitcoin_deposit_and_withdraw_with_dust.go [hunk 2]
```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // TestBitcoinDepositAndWithdrawWithDust deposits Bitcoin and call a smart contract that withdraw dust amount
```
Comment (lumtis) [inline e2e/e2etests/test_bitcoin_deposit_and_withdraw_with_dust.go]:
```suggestion
// TestBitcoinDepositAndWithdrawWithDust deposits Bitcoin and call a smart contract that withdraws dust amount
```

File: e2e/e2etests/test_bitcoin_deposit_and_withdraw_with_dust.go [hunk 3]
```diff
@@ -13,7 +13,7 @@ import (
...
-// It tests the edge case where during a cross-chain call, a invaild withdraw is initiated (processLogs fails)
+// It tests the edge case where during a cross-chain call, a invalid withdraw is initiated (processLogs fails)
```
Comment (lumtis) [inline e2e/e2etests/test_bitcoin_deposit_and_withdraw_with_dust.go]:
```suggestion
// It tests the edge case where during a cross-chain call, an invalid withdraw is initiated (processLogs fails)
```
